src/board.cpp

```diff
@@ -1,30 +1,33 @@
 #include "board.h"
 
+
 Board::Board() {
-    cout << "making a bored" << endl;
+    std::cout << "making a bored" << std::endl;
     this->moves = 0;
 }
 
 int Board::getMoves() {
     return this->moves;
 }
 
-Piece* Board::at(string pos) {
-    cout << "checking for duders at " << pos << endl;
+Piece* Board::at(std::string pos) {
+    std::cout << "checking for duders at " << pos << "\n";
     int x = pos[0];
-    int y = boost::lexical_cast<int>(pos[1]);
+    //int y = boost::lexical_cast<int>(pos[1]);
```

Comment:
As git will keep this in its history, feel free to whack these commented out lines.